webapp/src/pages/CreateChatRoomPage/CreateChatRoomPage.tsx

```diff
@@ -0,0 +1,190 @@
+      return { ...previousValues, [name]: value };
+    });
+  };
+
+  function hasValidFormValues() {
+    if (formValues.chatName === '') {
+      setStatus({
+        type: 'error',
+        message: 'ChatName can not be empty',
+      });
+      return false;
+    } else if (formValues.password !== formValues.confirmationPassword) {
+      setStatus({
+        type: 'error',
+        message: 'Passwords are different',
+      });
+      return false;
+    }
+    return true;
+  }
+  async function register() {
+    if (!hasValidFormValues()) {
+      return;
+    }
+    try {
+      console.log(formValues);
+      await chatsApi.chatControllerCreateChat({
+        createChatDto: {
+          ...formValues,
+          password: formValues.password || null,
+          confirmationPassword: formValues.confirmationPassword || null,
+        },
+      });
+      setStatus({
+        type: 'success',
+        message: 'Registration complete',
+      });
+      setTimeout(() => {
+        navigate(CHAT_URL);
+      }, 2000);
+    } catch (error) {
+      console.log(formValues);
+      if (error instanceof ResponseError) {
+        setStatus({ type: 'error', message: `${error.response.statusText}` });
+      } else if (error instanceof Error) {
+        setStatus({ type: 'error', message: `${error.message}` });
+      }
+    }
+  }
+  const handleOnSubmit = (e: React.FormEvent<HTMLFormElement>): void => {
+    e.preventDefault();
+    register().catch((e) => console.error(e));
+  };
+
+  return (
+    <div className="create-chat-page">
+      <Header icon={IconVariant.ARROW_BACK} onClick={goBack()}>
+        add chat
+      </Header>
+      <div className="create-chat-page-avatar-properties">
+        <div className="create-chat-page-avatar">
+          <MediumAvatar url={WILDCARD_AVATAR_URL} />
+        </div>
+        <div className="create-chat-page-properties">
+          <Text
+            variant={TextVariant.PARAGRAPH}
+            color={TextColor.LIGHT}
+            weight={TextWeight.REGULAR}
+          >
+            {formValues.chatName ? formValues.chatName : 'chat name'}
+          </Text>
+          <Text
+            variant={TextVariant.PARAGRAPH}
+            color={TextColor.LIGHT}
+            weight={TextWeight.REGULAR}
+          >
+            {formValues.password ? 'private channel' : 'public channel'}
+          </Text>
+        </div>
+      </div>
+      <form
+        id="create-chat-page-form"
+        className="create-chat-page-form"
+        onSubmit={handleOnSubmit}
+      >
+        <div className="create-chat-page-form-inputs-container">
+          <Input
+            variant={InputVariant.LIGHT}
+            label="ChatName"
+            placeholder="chat name"
+            value={formValues.chatName}
+            name="chatName"
+            onChange={handleInputChange}
+          />
+          <Input
+            variant={InputVariant.LIGHT}
+            label="Password"
```

Comment:
As of now users can write in private chats where they don't input the password. Is this something that is supposed to work?

Reply:
No, not yet. That something left to do, I'm going to check if there is a ticket about it and, if not, I will create